presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -219,10 +263,10 @@ private void runTests(Environment environment)
 
                 InspectContainerResponse containerInfo = container.getCurrentContainerInfo();
                 ContainerState containerState = containerInfo.getState();
-                Integer exitCode = containerState.getExitCode();
+                Long exitCode = containerState.getExitCodeLong();
```

Comment:
Why was this needed?

Reply:
No. I've just seen a deprecated method use so I replaced it with non-deprecated one :)